test/e2e/run.test.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="jest-playwright-preset" />
+
+import {
+  makeAudienceDefinition,
+  makeStringArrayQuery,
+} from '../helpers/audienceDefinitions';
+
+type Store = { edkt_matched_audiences: string; edkt_page_views: string };
+
+const getPageViewsFromStore = (store: Store) =>
+  JSON.parse(store['edkt_page_views']);
+const getMatchedAudiencesFromStore = (store: Store) =>
+  JSON.parse(store['edkt_matched_audiences']);
+const getLocalStorageFromPage = (): Promise<Store> =>
+  page.evaluate('localStorage');
+
+describe('edgekit basic run behaviour', () => {
+  // We are serving a mock page with the edgekit transpiled code.
+  // look at jest-playwright.config.js
+  const testUrl = 'http://localhost:9000';
+
+  const sportAudience = makeAudienceDefinition({
+    id: 'sport_id',
+    occurrences: 1,
+    definition: [makeStringArrayQuery(['sport'])],
```

Comment:
We have not been using string array - most likely we should remove it.

Would you mind updating this to test cosine sim audience please?

Reply:
Not at all! I'll update it!

Reply:
legend